examples/hooks/components/SearchBox.tsx

```diff
@@ -19,8 +19,14 @@ export function SearchBox(props?: SearchBoxProps) {
   }
 
   useEffect(() => {
-    refine(value);
-  }, [refine, value]);
+    if (query !== value) {
```

Comment:
we are relying on those both being `null` (or '') initially, right?

Reply:
Yes, `value` is initialized with InstantSearch's `query` ([which is an empty string initially](https://github.com/algolia/instantsearch.js/blob/29fafcaa60a0b004e808a5187bed47beca47e703/src/connectors/search-box/connectSearchBox.ts#L162)).